.generator/src/generator/templates/modelSimple.j2

```diff
@@ -255,6 +255,47 @@ public class {{ name }} {%- if model.get("x-generate-alias-as-model") %} extends
   }
   {%- endif %}
   {%- endfor %}
+  {%- if model.get("additionalProperties") %}
```

Comment:
imo the default should be true

Reply:
Yeah I get your point, but I'd like to get more understanding about the use before adding 10K lines of code.